packages/manager/apps/dedicated/client/app/dedicated/server/translations/Messages_fr_FR.json

```diff
@@ -1166,6 +1166,7 @@
   "server_close_service_warning_manual_refund": "L'action de suppression ne garantie pas l'octroi d'un remboursement. Contactez le support si vous avez des questions concernant un éventuel remboursement.",
   "server_close_service_desc": "Un email contenant la procédure de suppression de ce serveur vous sera envoyé.",
   "server_close_service_question": "Voulez-vous continuer ?",
+  "server_close_service_enterprise": "Pour obtenir de l'aide pour apporter des modifications contractuelles à vos services, veuillez contacter votre responsable de compte.",
```

Comment:
It this a validated content?

Reply:
The english version of it is validated. I generated the FR version by doing a manual translation. I am yet to get the validated FR version (JIRA already updated, requesting for the correct FR version). Thanks

Reply:
The FR text has been updated. Thanks!